src/pipeline/src/etl/processor/cmcd.rs

```diff
@@ -201,28 +206,51 @@ impl Default for CmcdOutputInfo {
 }
 
 /// function to resolve CMCD_KEY_BS | CMCD_KEY_SU
-fn bs_su(_: &str, _: &str, _: Option<&str>) -> Result<Value, String> {
+fn bs_su(_: &str, _: &str, _: Option<&str>) -> Result<Value> {
     Ok(Value::Boolean(true))
 }
 
 /// function to resolve CMCD_KEY_BR | CMCD_KEY_BL | CMCD_KEY_D | CMCD_KEY_DL | CMCD_KEY_MTP | CMCD_KEY_RTP | CMCD_KEY_TB
-fn br_tb(s: &str, k: &str, v: Option<&str>) -> Result<Value, String> {
-    let v = v.ok_or(format!("{k} missing value in {s}"))?;
-    let val: i64 = v
-        .parse()
-        .map_err(|_| format!("failed to parse {v} as i64"))?;
+fn br_tb(s: &str, k: &str, v: Option<&str>) -> Result<Value> {
+    let v = v.ok_or(
```

Comment:
after import `snafu::OptionExt`, it also possible to do 

v.context(CmcdMissingValueSnafu {... })